src/main/java/ru/javarush/quest/impl/AnswerImpl.java

```diff
@@ -0,0 +1,62 @@
+package ru.javarush.quest.impl;
+
+import ru.javarush.quest.entity.Answer;
+import ru.javarush.quest.service.AbstractDao;
+import ru.javarush.quest.service.AnswerDao;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class AnswerImpl extends AbstractDao implements AnswerDao {
+    @Override
+    public Answer getAnswersInOrder(int count) {
+
+        Answer answer = null;
+        try (Connection connection = connect();
+             Statement stmt = connection.createStatement()) {
+            stmt.execute("SELECT * from answers where id=" + count);
+            ResultSet resultSet = stmt.getResultSet();
+
+            while (resultSet.next()) {
+                answer = (getAnswer(resultSet));
+            }
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+
+        return answer;
+    }
+
+    @Override
+    public String wrongAnswerDescription(int count) {
+        String wrongAnswerDescription = null;
+        try (Connection connection = connect();
+             Statement stmt = connection.createStatement()) {
+            stmt.execute("SELECT d.description FROM description_for_wrong_answer d LEFT JOIN answers " +
+                    "ON answers.id = d.id where d.id = " + count);
+            ResultSet rs = stmt.getResultSet();
+            while (rs.next()) {
+                wrongAnswerDescription = getWrongAnswerDescription(rs);
+            }
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
It is ok just log exception and return null?

Reply:
What do you here. i didnt get it